src/main/java/runge/worldcities/WorldCitiesTests.java

```diff
@@ -0,0 +1,31 @@
+package runge.worldcities;
```

Comment:
This file needs to be under `src/test/java`

Reply:
I tried to do that but got very wonky